cli/dev_pipeline.py

```diff
@@ -48,6 +48,7 @@ def run_on_document(document_path: Path):
             ),
             chunk_processors.SplitTextIntoSentences(),
             chunkers.FixedLengthChunker(max_chunk_words=150),
+            tokenizers.NLTKWordTokenizer(),
```

Comment:
i know i keep harping on about it, but i love how easy it is to read this pipeline stuff! makes me v happy to see so much complexity described so neatly